app/jobs/subscriptions/organization_billing_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Subscriptions
+  class OrganizationBillingJob < ApplicationJob
+    queue_as :default
+
+    def perform(organization:)
+      Subscriptions::OrganizationBillingService.call!(organization:)
```

Comment:
and:
```suggestion
      Organizations::BillSubscriptionsService.call!(organization:)
```

Reply:
I though about it, but since the main entities we are dealing with in the service are subscriptions and not the organization (just a filter), I decided to keep the namespace